### PR TITLE
Palette: ColorblindFriendly

### PR DESCRIPTION
**Pull request by arthurits**
**New Contributors:**
Please review [CONTRIBUTING.md](https://github.com/swharden/ScottPlot/blob/master/CONTRIBUTING.md)

**Hacktoberfest Participants:**
Check-out the [Hacktoberfest 2021](https://github.com/ScottPlot/ScottPlot/issues/1274) page

**Purpose:**
* Add a new 8-color palette that has good overall variability and can be differentiated by individuals with red-green color blindness, as suggested by [Bang Wong (Nature Methods, 2011)](https://www.nature.com/articles/nmeth.1618.pdf).
* This is one of the many subjects suggested in #1274.
 
image

### CODE REVIEW

**Comment by swharden** [comment]
Thanks @arthurits! As someone with red-green colorblindness, I very much appreciate this PR 😄

I'll merge this (and the adjacent ones) after @bclehmann approves 👍

**Review by swharden — APPROVED** [review]

**Comment by arthurits** [comment]
> Thanks @arthurits! As someone with red-green colorblindness, I very much appreciate this PR 😄

My pleasure!

File: src/ScottPlot/Palette.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public static class Palette
         public static ScottPlot.Drawing.Palette Aurora => new(new ScottPlot.Drawing.Colorsets.Aurora());
         public static ScottPlot.Drawing.Palette Category10 => new(new ScottPlot.Drawing.Colorsets.Category10());
         public static ScottPlot.Drawing.Palette Category20 => new(new ScottPlot.Drawing.Colorsets.Category20());
+        public static ScottPlot.Drawing.Palette ColorBlindness => new(new ScottPlot.Drawing.Colorsets.ColorblindFriendly());
```
Comment (bclehmann) [inline src/ScottPlot/Palette.cs:17]: I'm wondering if `ColorBlind` or `ColorBlindFriendly` is a better name than `ColorBlindness`? I'm not sure how to articulate why. Either name is fine with me, so go with whichever you prefer.

**Review by bclehmann — APPROVED** [review]
LGTM!

File: src/ScottPlot/Palette.cs (hunk 1 above)
Comment (swharden) [reply src/ScottPlot/Palette.cs:17]: Oops, good catch! I'll make this change and merge. Thanks!

**Review by swharden — APPROVED** [review]

File: src/cookbook/ScottPlot.Cookbook/Recipes/Palette.cs [hunk 2]
```diff
@@ -273,16 +273,18 @@ public void ExecuteRecipe(Plot plt)
         }
     }
 
-    public class PaletteColorBlindness : IRecipe
+    public class PaletteColorblindFriendly : IRecipe
     {
         public string Category => "Palette";
         public string ID => "palette_ColorBlindness";
         public string Title => "ColorBlindness";
```
Comment (bclehmann) [inline src/cookbook/ScottPlot.Cookbook/Recipes/Palette.cs:280]: You missed the title

Comment (swharden) [reply src/cookbook/ScottPlot.Cookbook/Recipes/Palette.cs]: Great catch! Thanks for looking over my work so carefully 😄

**Pull request merged**